terra_notebook_utils/workflows.py

```diff
@@ -83,19 +83,27 @@ def estimate_workflow_cost(workflow_id: str, workflow_metadata: dict) -> Generat
                 task_name = call_name.split(".")[1]
                 call_cached = bool(int(js_get("callCaching.hit", call_metadata)))
                 if call_cached:
-                    cost, cpus, memory_gb, runtime = 0.0, 0, 0.0, 0.0
+                    cost, cpus, memory_gb, runtime, disk_size_gb = 0.0, 0, 0.0, 0.0, 0.0
                 else:
                     cpus, memory_gb = _parse_machine_type(js_get("jes.machineType", call_metadata))
                     # Assume that Google Lifesciences Pipelines API uses N1 custome machine type
                     start = datetime.strptime(js_get("start", call_metadata), date_format)
                     end = datetime.strptime(js_get("end", call_metadata), date_format)
                     runtime = (end - start).total_seconds()
                     preemptible = bool(int(js_get("runtimeAttributes.preemptible", call_metadata)))
-                    cost = costs.GCPCustomN1Cost.estimate(cpus, memory_gb, runtime, preemptible)
+                    disk_description = js_get("runtimeAttributes.disks", call_metadata, default="")
+                    if disk_description.startswith("local-disk"):
+                        _, size_gb, _ = disk_description.split()
+                        disk_size_gb = float(size_gb)
+                    else:
+                        disk_size_gb = 1.0
```

Comment:
Not clear why this is set to ```1.0```... Maybe I'm missing some context?

Reply:
I’ve added a comment which hopefully clears things up:
```
disk_size_gb = 1.0  # Guess 1GB when disk information is unavailable
```